crates/sncast/tests/helpers/constants.rs

```diff
@@ -10,7 +10,7 @@ pub const NETWORK: &str = "testnet";
 pub const SEED: u32 = 1_053_545_548;
 
 // Block number used by devnet to fork the Sepolia testnet network in the tests
-pub const FORK_BLOCK_NUMBER: u32 = 68_000;
+pub const FORK_BLOCK_NUMBER: u32 = 100_000;
```

Comment:
That's because of the data transformer contract - we need it in our fork of Sepolia. It's been deployed later in the chain so I bumped the block number.